pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -1,59 +1,99 @@
 { stdenv
 , lib
 , fetchFromGitHub
+, fetchFromGitLab
 , fetchpatch
 , fetchurl
-, pkg-config
+, Foundation
+, abseil-cpp
 , cmake
-, python3Packages
 , libpng
-, zlib
-, eigen
-, protobuf
-, howard-hinnant-date
 , nlohmann_json
-, boost
-, oneDNN_2
-, abseil-cpp
-, gtest
-, pythonSupport ? false
 , nsync
-, flatbuffers
+, pkg-config
+, python3Packages
+, re2
+, zlib
+, microsoft-gsl
+, microsoft-wil
```

Comment:
To my embarrassment, it appears that `microsoft-wil` is not needed after all to build `onnxruntime`